src/renderers/shared/fiber/__tests__/ReactIncrementalUpdates-test.js

```diff
@@ -154,19 +154,20 @@ describe('ReactIncrementalUpdates', () => {
     expect(ReactNoop.getChildren()).toEqual([span('')]);
 
     // Schedule some more updates at different priorities{
-    instance.setState(createUpdate('f'));
+    instance.setState(createUpdate('d'));
     ReactNoop.flushSync(() => {
-      instance.setState(createUpdate('d'));
       instance.setState(createUpdate('e'));
+      instance.setState(createUpdate('f'));
     });
     instance.setState(createUpdate('g'));
 
     // The sync updates should have flushed, but not the async ones
-    expect(ReactNoop.getChildren()).toEqual([span('de')]);
+    expect(ReactNoop.getChildren()).toEqual([span('ef')]);
 
-    // Now flush the remaining work. Updates a, b, and c should be processed
-    // again, since they were interrupted last time.
-    expect(ReactNoop.flush()).toEqual(['a', 'b', 'c', 'f', 'g']);
+    // Now flush the remaining work. Even though e and f were already processed,
+    // they should be processed again, to ensure that the terminal state
+    // is deterministic.
+    expect(ReactNoop.flush()).toEqual(['a', 'b', 'c', 'd', 'e', 'f', 'g']);
     expect(ReactNoop.getChildren()).toEqual([span('abcdefg')]);
   });
```

Comment:
I'm a little confused here, even before this PR, the the terminal state is still deterministic(i.e. `ReactNoop.getChildren()` === `abcdefg `). Seems this PR only change the behavior of `ReactNoop.flush()`? But I don't know why the it just change the `ReactNoop.flush()` rather than `ReactNoop.getChildren()`, the React relays on flush so I think it should have same result of `flush` and `getChildren`